content/members.md

```diff
@@ -55,33 +50,20 @@ You can contact us at contact(at)fink-broker.org for more information.
 
 * **CEICO, Institute of Physics, Czech Academy of Sciences**: S. Karpov
 
-#### Denmark
-
-* **Dark Cosmology Centre (DARK), Niels Bohr Institute - University of Copenhagen**: C. Gall
-
-#### Italy
-
-* **Istituto Nazionale di Fisica Nucleare – Sezione di Genova**: V. Kulikovskiy
-
 #### France
 
 * **CEA/AIM, Université Paris-Saclay**: D. Turpin
 * **CEA/Irfu, Université Paris-Saclay**: F. Schüssler
-* **CNRS/APC, Université de Paris**: B. Biswas, A. Boucaud, A. Coleiro
 * **CNRS/CC-IN2P3**: F. Hernandez
 * **CNRS/CDS, Observatoire Astronomique de Strasbourg, Université de Strasbourg**: A. Nebot Gomez-Moran
-* **CNRS/CPPM, Aix Marseille Université**: D. Dornic, D. Fouchez, G. Vannoye
-* **CNRS/IAP, Sorbonne Université**: F. Daigne, J.G. Ducoin
-* **CNRS/IJCLab, Université Paris-Saclay**: R. Ansari, J.-E. Campagne, R. Le Montagner, N. Leroy, M. Moniez, J. Neveu
-* **CNRS/IP2I, Université Claude Bernard Lyon 1**: M. Rigault, M. Smith
-* **CNRS/IRAP, Université Paul Sabatier**: O. Godet, M.L. Lanza, N. Webb, M. Yassine, H. Yang
+* **CNRS/CPPM, Aix Marseille Université**: D. Fouchez
+* **CNRS/IJCLab, Université Paris-Saclay**: R. Le Montagner, N. Leroy, M. Moniez, J. Neveu
+* **CNRS/IRAP, Université Paul Sabatier**: M.L. Lanza, N. Webb, H. Yang
 * **CNRS/LAM, Aix Marseille Université**: S. Blondin
-* **CNRS/LAPP, Université Grenoble-Alpes, Université Savoie Mont Blanc**: D. Boutigny, D. Verkindt
-* **CNRS/LLR**: H. Ashkar
-* **CNRS/LPCA, Université Clermont Auvergne**: E. Gangler, C. Mur, P. Gris
+* **CNRS/LAPP, Université Grenoble-Alpes, Université Savoie Mont Blanc**: D. Boutigny
+* **CNRS/LPCA, Université Clermont Auvergne**: J. Cohen-Tanugi, E. Gangler, C. Mur
 * **CNRS/LPSC, Université Grenoble-Alpes**: J. Bregeon, M. Masson
-* **CNRS/LUPM, Université de Montpellier**: J. Cohen-Tanugi
-* **CNRS/OCA, Lagrange, Université Côte d'Azur**: B. Carry, A. Sergeyev, K. O. Xenos
+* **CNRS/OCA, Lagrange, Université Côte d'Azur**: B. Carry
```

Comment:
K. O. Xenos should be here (sent his form for the first time few days ago)

Reply:
added